packages/combobox/src/Combobox.ts

```diff
@@ -117,7 +104,7 @@ export class Combobox extends Textfield {
             this.open = true;
             this.activateNextDescendant();
             const activeEl = this.querySelector(
-                `#${(this.activeDescendant as ComboboxOption).id}`
+                `#${(this.activeDescendant as ComboboxOption).value}`
```

Comment:
Is the type assertion still necessary, now that properties align with `MenuItem`?

Reply:
Good call! It was a force type, not a change type, but I merged the two uses and went to `this.activeDescendat?.id` instead of the type application.